react/features/calendar-sync/components/AbstractCalendarList.js

```diff
@@ -221,39 +182,56 @@ class MeetingList extends Component<Props> {
     _toDisplayableList() {
         const { _eventList, t } = this.props;
 
-        const now = Date.now();
+        const now = new Date();
 
         const { createSection } = NavigateSectionList;
-        const nowSection = createSection(t('calendarSync.now'), 'now');
-        const nextSection = createSection(t('calendarSync.next'), 'next');
-        const laterSection = createSection(t('calendarSync.later'), 'later');
+        const TODAY_SECTION = 'today';
+        const todaySection
+            = createSection(t('calendarSync.today'), TODAY_SECTION);
+        const sectionMap = new Map();
+
+        sectionMap.set(TODAY_SECTION, todaySection);
```

Comment:
I think this means sections will always have a length of at least 1, which means SectionList will never show ListEmptyComponent.